students/seoyun/westagram/users/views.py

```diff
@@ -53,9 +54,10 @@ def post(self, request):
             password   = user_data['password']
             login_user = User.objects.get(email=email)
             
-            if not (password == login_user.password):
-                return JsonResponse({"message": "INVALID_USER_PASSWORD"}, status=401)
-            return JsonResponse({'message':'SUCCESS'}, status=201)
+            if bcrypt.checkpw(password.encode('utf-8'), login_user.password.encode('utf-8')):
+                token = jwt.encode({'user-id': login_user.id}, settings.SECRET_KEY, settings.ALGORITHM)
+                return JsonResponse({'token': token}, status=201)
+            return JsonResponse({"message": "INVALID_USER_PASSWORD"}, status=401)
```

Comment:
서윤님! 현재 작성하신 코드의 경우 
- if 문으로 성공케이스를 거르고
- 실패시 error msg 리턴

구조를 갖고 있습니다. 

이렇게 성공 케이스를 먼저 거르는 방식으로 코드를 작성하면 if 문으로 처리해줘야 하는 로직이 많아질 경우 굉장히 복잡해질 수 있습니다. 따라서 논리의 흐름대로 `문제가 있을만한 경우를 if문으로 거르고 모두 통과시 성공한다` 식으로 코드를 작성해주면 더 좋을 것 같습니다.

```suggestion
            if not bcrypt.checkpw(password.encode('utf-8'), login_user.password.encode('utf-8')):
                return JsonResponse({"message": "INVALID_USER_PASSWORD"}, status=401)
            
            if not ...:
                return ....
                
            if not ....:
                return ....
            
            token = jwt.encode({'user-id': login_user.id}, settings.SECRET_KEY, settings.ALGORITHM)
            return JsonResponse({'token': token}, status=201)

```

Reply:
추가로 토큰의 경우 그 의미를 담고 있는 `access_token`이라고 명확하게 적는 것이 더 좋아보입니다.

Reply:
넵 수정완료했습니다!